python/ray/_raylet.pxd

```diff
@@ -150,7 +150,7 @@ cdef class CoreWorker:
             worker, outputs,
             const CAddress &caller_address,
             c_vector[c_pair[CObjectID, shared_ptr[CRayObject]]] *returns,
-            CObjectID ref_generator_id=*)
+            ref_generator_id=*)
```

Comment:
is this a workaround for the compiler bug you mentioned? if so let's at least leave something like:
```suggestion
            ref_generator_id=*, # CObjectID
        )
```

Reply:
Also what does `=*` mean here? Not familiar with the syntax

Reply:
This is the way to signal an optional value in a pxd (header) file -- cython needs to know since it creates the C signature based on it. See https://cython.readthedocs.io/en/latest/src/userguide/language_basics.html#optional-arguments :)